consensus/src/pipeline/buffer_item.rs

```diff
@@ -40,18 +40,18 @@ fn generate_commit_ledger_info(
     )
 }
 
-fn ledger_info_with_unverified_signatures(
+fn signature_aggregator(
```

Comment:
nit: the name is not very descriptive, create_ or construct_signature_aggregator